### PR TITLE
Uitpas/ups 3504 extra

### PR DESCRIPTION
**Pull request by erwin1**
### Changed

- fixed array type for city and postalCode
- documented 404
- added allowsCardlessRegistration boolean

---

Ticket: https://jira.uitdatabank.be/browse/UPS-3504

### CODE REVIEW

File: projects/uitpas/reference/uitpas.json [hunk 1]
```diff
@@ -4378,22 +4378,32 @@
             },
             "description": "Maximum amount of results to return. Can be used in combination with `start` for pagination.",
             "name": "limit"
-          },
+          },
```
Comment (bertramakers) [inline projects/uitpas/reference/uitpas.json]: Nit: These trailing whitespaces can be removed

Comment (bertramakers) [reply projects/uitpas/reference/uitpas.json]: Ah resolved in the next commit

File: projects/uitpas/reference/uitpas.json [hunk 2]
```diff
@@ -4352,6 +4352,9 @@
           },
           "403": {
             "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Not Found. Error type:\n\nhttps://api.publiq.be/probs/url/not-found"
```
Comment (bertramakers) [inline projects/uitpas/reference/uitpas.json]: In which case would this endpoint return a 404? If there are no results found I'd expect it to return 200 but with an empty list.

**Review by bertramakers — APPROVED** [review]
Not 100% sure when `GET /card-systems` would return a 404, but other than that it looks good!

File: projects/uitpas/reference/uitpas.json (hunk 2 above)
Comment (erwin1) [reply projects/uitpas/reference/uitpas.json]: this can be the case when postalCode or city does not validate

**Pull request merged**